### ISSUE TITLE
These should have an aria-label

### ISSUE THREAD

**Issue by ciaoaaron** [jshulmartin-stack/Desktop-Library-Website#5]
https://github.com/jshulmartin-stack/Desktop-Library-Website/blob/8ba667586771c5efa3f8703e76d9fa815c02269a/index.html#L62-L64

A screen reader user would only hear "Link" and not knowing what they are for. One way to make it better would be to add an `aria-label` like this:

```
<a href="#slide-1" aria-label="slide 1"></a>

```
